src/mqueryfront/src/QueryResultsStatus.js

```diff
@@ -44,35 +41,53 @@ function MatchItem(props) {
         ));
     }
 
-    const download_url =
-        API_URL +
-        "/download?job_id=" +
-        encodeURIComponent(props.qhash) +
-        "&ordinal=" +
-        encodeURIComponent(props.ordinal) +
-        "&file_path=" +
-        encodeURIComponent(props.file);
-
-    const path = require("path");
-
     return (
         <tr>
             <td>
                 <div className="row m-0 text-truncate">
                     <div className="text-truncate" style={{ minWidth: 50 }}>
-                        <a
-                            href={download_url}
-                            data-toggle="tooltip"
-                            title={props.file}
-                        >
+                        <div>
+                            {props.meta.sha256.display_text}
+                            <small>
+                                <a
+                                    href={download_url}
+                                    data-toggle="tooltip"
+                                    title={props.file}
+                                    className="text-secondary"
+                                >
+                                    <i className="fa fa-download fa-xm ml-2" />
+                                </a>
+                                <CopyToClipboard
+                                    text={props.meta.sha256.display_text}
+                                >
+                                    <span
+                                        data-toggle="tooltip"
+                                        title="Copy sha256 to clipboard"
```

Comment:
I *think* it should be `Copy sha256 to the clipboard`, but both forms are used on the internet so correct me if I'm wrong.

```suggestion
                                        title="Copy sha256 to the clipboard"
```

Reply:
Actually that would imply, that there could be multiple clipboards (so you need to specify which one to copy to). *Copy sha256 to clipboard* is perfectly fine.

Reply:
`Copy to clipboard` is popular enough on the internet that you're probably right, I was just a bit confused because of the number of counter-examples on the internet, for example:

- [mozilla docs](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard) - `Writing to the clipboard`, `copying the current selection into the clipboard` etc
- [office support](https://support.office.com/en-us/article/use-the-office-clipboard-028903c7-66ea-4eb0-b0a1-bacdb952e3a1) - `Cut and paste items from the Clipboard`, `To paste everything in the Clipboard`
- [wikipedia](https://en.wikipedia.org/wiki/Clipboard_(computing)) -  all `clipboards` prefixed with `the`
- [random tech site](https://www.digitalcitizen.life/simple-questions-what-clipboard-windows-how-manage-it) - `copy to the clipboard overwrites the previous contents of the clipboard`
- [w3schools](https://www.w3schools.com/howto/howto_js_copy_clipboard.asp) is bipolar on this, because post title is `How TO - Copy Text to Clipboard` and the description is `Learn how to copy text to the clipboard with JavaScript`.

Intuitively your explaination makes sense though, so i guess nevermind my comment here :+1:

Reply:
Well, english is hard. For example you have *Sun goes around **the** world*. You don't use *the* to specify which world, but there's only 1 (and it's obvious). However, if the context isn't as obvious (as clipboard) skipping *the* can be used to imply that there's only one clipboard. 

My guess that both versions are correct. In theory you should add *the*, but by omitting you provide more information to user.

Reply:
Thanks, you two <3